packages/commonwealth/client/scripts/views/components/Polls/PollOptions/PollOptions.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { CWCheckbox } from 'views/components/component_kit/cw_checkbox';
+import { CWRadioButton } from 'views/components/component_kit/cw_radio_button';
+
+import './PollOptions.scss';
+
+export type VoteInformation = {
+  label: string;
+  value: string;
+  voteCount: number;
+};
+
+export type PollOptionProps = {
+  multiSelect: boolean;
+  voteInformation: Array<VoteInformation>;
+  selectedOptions?: Array<string>;
+  disableVoteOptions?: boolean;
+  setSelectedOptions?: React.Dispatch<React.SetStateAction<string[]>>;
+};
+
+export const PollOptions = ({
+  disableVoteOptions,
+  multiSelect,
+  selectedOptions,
+  voteInformation,
+  setSelectedOptions,
+}: PollOptionProps) => {
+  return (
+    <div className="PollOptions">
+      {multiSelect
+        ? voteInformation.map((option) => (
+            <CWCheckbox
+              checked={false}
+              value=""
+              label={option.label}
+              key={option.value}
+              onChange={() => {
+                // TODO: Build this out when multiple vote options are introduced.
+                // Something like: selectedOptions.push(option.value);
+                console.log('A vote for multiple options');
+              }}
+            />
+          ))
+        : voteInformation.map((option) => (
+            <CWRadioButton
+              key={option.value}
+              checked={
+                // @ts-expect-error <StrictNullChecks/>
+                selectedOptions.length > 0 &&
```

Comment:
lets use `?.` instead of `// @ts-expect-error <StrictNullChecks/>`, which won't throw an error if `selectedOptions` is renamed/type-changed

Reply:
thanks for letting me know, changed